src/WCSimSteppingAction.cc

```diff
@@ -100,7 +100,7 @@ G4int WCSimSteppingAction::G4ThreeVectorToWireTime(G4ThreeVector *pos3d,
 } 
 
 
-void WCSimSteppingAction::Distortion(G4double x,G4double y)
+void WCSimSteppingAction::Distortion(G4double /*x*/,G4double /*y*/)
```

Comment:
Ok, I'm not looking to make wasted work for anyone, and we will have the next PR to focus on once this gets merged, so let's leave these compiler warning changes in this PR. But, for future pull requests here is what the philosophy should be: the changes that get made in the PR should be directly related to the issue or new feature that is being implemented. So, if you had found that there were compiler warnings, you would make a pull request just for that, we would have had a little discussion about what the warnings were and how you fixed them, and it would have been merged in awhile ago. Then, in the future, if I was wondering why this line was like it is, I would click the blame button, see which pull request that line was last changed in, and would see our discussion on just that topic rather than having to wade through the long digitizer discussion like is the case now. This is a small change that probably won't cause many questions in the future, but it's good to get in the habit of having focused pull requests and I think it makes for easier bookkeeping if nothing else.